src/styles/MermaidStyledComponents.js

```diff
@@ -27,7 +32,7 @@ export const ButtonStyle = styled('button')`
     '0px 1px 3px 0px rgba(0,0,0,0.2), 0px 1px 1px 0px rgba(0,0,0,0.14), 0px 2px 1px -1px rgba(0,0,0,0.12)'};
   transform: ${props => props.theme.initialTranslate};
   transition: 0.2s ease-out;
-  cursor: ${props => (props.notAllowed ? 'not-allowed' : 'pointer')};
+  cursor: ${props => (props.notAllowed ? 'not-allowed' : props.disabled ? 'default' : 'pointer')};
```

Comment:
can you make a small change from 'default` to 'not-allowed`?

Reply:
Sure. The reason behind the cursor being `default` was to match the zoom in/out buttons' behavior of disabled, since they are in the same group of map control buttons.

Reply:
Got it! Consistency is great. I'm fine with that.